main/src/main/scala/sbt/internal/DefaultBackgroundJobService.scala

```diff
@@ -209,7 +209,6 @@ private[sbt] abstract class AbstractBackgroundJobService extends BackgroundJobSe
     }
     val xs = (products.toVector map { syncTo(workingDirectory / "target") }) ++
       ((full diff products) map { syncTo(serviceTempDir / "target") })
-    Thread.sleep(100)
```

Comment:
Looks like Thread.sleep(100) has been around since https://github.com/sbt/sbt/pull/2936.
LGTM if it passes the CI.

Reply:
Yeah. I figured if anyone would know the purpose of the sleep, it'd be you :)